src/toplevel/learnocaml_toplevel_output.ml

```diff
@@ -180,6 +180,36 @@ let output_html ?phrase output html =
   Js_utils.Manip.appendChild output.container div ;
   insert output ?phrase (Html (html, div)) div
 
+let get_fresh_id =
+  let r = ref 0 in
+  fun () -> incr r ; !r
+
+(* It replaces markup field id by "id-<number>" to avoid interferences
+   between svg image. *)
```

Comment:
```suggestion
(* It replaces markup field id by "id-<number>" to avoid interferences 
   between svg images when they are inserted in the same DOM. 
   In other words, we ensure that every identifier is unique. *)
```